src/Features/Onboarding.php

```diff
@@ -583,7 +583,7 @@ public static function has_woocommerce_support( $theme ) {
 			$files     = new \RegexIterator( $iterator, '/^.+\.php$/i', \RecursiveRegexIterator::GET_MATCH );
 
 			foreach ( $files as $file ) {
-				$content = file_get_contents( $file[0] );
+				$content = file_get_contents( $file[0] ); // phpcs:ignore WordPress.WP.AlternativeFunctions
```

Comment:
This call and the changes on line 978 and 1021 where from the php linter, I couldn't commit otherwise.

Reply:
I think https://codex.wordpress.org/Filesystem_API is what you should be using rather than `file_get_contents` directly

Reply:
Meaning something like this?
```
global $wp_filesystem;
$wp_filesystem->get_contents()
```
I just added the phpcs comment to fix a pre-commit error.

Reply:
I decided to revert this change, as it wasn't necessary anymore (no pre-commit hook) and unrelated to any of the changes of this PR.